packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -409,10 +409,70 @@ class FlutterDriver {
 
   /// Scrolls the Scrollable ancestor of the widget located by [finder]
   /// until the widget is completely visible.
+  ///
+  /// If the widget located by [finder] is contained by a scrolling widget
+  /// that lazily creates its children, like [ListView] or [CustomScrollView],
+  /// then this method may fail because [finder] doesn't actually exist.
+  /// The [scrollUntilVisible] method can be used in this case.
   Future<Null> scrollIntoView(SerializableFinder finder, { double alignment: 0.0, Duration timeout }) async {
     return await _sendCommand(new ScrollIntoView(finder, alignment: alignment, timeout: timeout)).then((Map<String, dynamic> _) => null);
   }
 
+  /// If widget located by [item] is not visible then repeatedly [scroll] the
+  /// widget located by [scroller] by [dxScroll] and [dyScroll] until [item]
+  /// is visible, and then use [scrollIntoView] to ensure the item's final
+  /// position matches [alignment].
+  ///
+  /// The [scroller] must locate the scrolling widget that contains [item].
+  /// Typically `find.byType('ListView') or `find.byType('CustomScrollView')`.
+  ///
+  /// If [item] is below the currently visible items, then specify a negative
+  /// value for [dyScroll] that's a small enough increment to expose [item]
+  /// without potentially scrolling it up and completely out of view. Similarly
+  /// if [item] is above, then specify a positve value for [dyScroll].
+  ///
+  /// If [item] is to the right of the the currently visible items, then
+  /// specify a negative value for [dxScroll] that's a small enough increment to
+  /// expose [item] without potentially scrolling it up and completely out of
+  /// view. Similarly if [item] is to the left, then specify a positve value
+  /// for [dyScroll].
+  ///
+  /// The [timeout] value should be long enough to accomodate as many scrolls
+  /// as needed to bring an item into view. The default is 10 seconds.
+  Future<Null> scrollUntilVisible (SerializableFinder scroller, SerializableFinder item, {
```

Comment:
nit: for consistency let's refer to the scroller as "scrollable"
nit: remove space before `(`

Reply:
Done